src/EloquentHashids.php

```diff
@@ -51,4 +51,12 @@ public static function getHashidEncodingValue(Model $model)
   {
     return $model->getKey();
   }
+  
+  /**
+   * @return string
+   */
+  public function getRouteKeyName()
+  {
+      return 'uid';
```

Comment:
thanks - I will look at it this week. It's a bit much with all this christmas right now :)